StRoot/StFwdTrackMaker/include/Tracker/ObjExporter.h

```diff
@@ -0,0 +1,443 @@
+                // sphere( TVector3( p.X() * SCALE, p.Y() * SCALE, -p.Z() * SCALE ), 0.75, 10, 10, ofile );
+                TVector3 ds = TVector3( 1, 1, fcsClusterEnergy[i] );
+                prism( ofile, TVector3( p.X() * SCALE, p.Y() * SCALE, -p.Z() * SCALE ), ds );
+                i++;
+            }
+        }
+        
+        // Write the track seeds
+        if (verbose){
+            LOG_INFO << "Viz has " << seeds.size() << " seeds" << endm;
+        }
+        if ( seeds.size() > 0 ){
+            ofile << "\n\no FwdSeeds\n" << endl;
+            ofile << "usemtl seeds\n" << endl;
+            // numVertices = 0;
+            for ( auto s : seeds ) {
+                size_t vStart = numVertices;
+                for ( auto h : s ){
+
+                    vert( ofile, h->getX() * SCALE, h->getY() * SCALE, -h->getZ() * SCALE );
+
+                }
+
+                ofile << "l ";
+                for ( size_t i = vStart; i < numVertices; i++){
+                    ofile << i+1 << " "; 
+                }
+                ofile << endl;
+            }
+        }
+
+        // uncomment if you want a separate file for tracks
+        // ofile.open( (filename + "_tracks.obj" ).c_str()  );
+        // numVertices = 0;
+        // EXPORT TRACKS
+
+        if (verbose){
+            LOG_INFO << "Viz has " << tracks.size() << " tracks Hits" << endm;
+        }
+        if ( tracks.size() > 0 ){
+            ofile << "\n\no FwdTracks\n" << endl;
+            ofile << "usemtl tracks\n" << endl;
+            float zStep = 5.0; // cm
+            for ( auto t : tracks ) {
+                size_t vStart = numVertices;
+    
+                if ( false ) {
+    
+                    TVector3 point;
+                    point = startPos;
+                    vert( ofile, point.X() * SCALE, point.Y() * SCALE, -point.Z() * SCALE );
+    
+                    point = trackPosition( t, 281.0 );
+                    if ( point.X() > -90 && point.Y() > -90 ) 
+                        vert( ofile, point.X() * SCALE, point.Y() * SCALE, -point.Z() * SCALE );
+    
+                    point = trackPosition( t, 304.0 );
+                    if ( point.X() > -90 && point.Y() > -90 )
+                        vert( ofile, point.X() * SCALE, point.Y() * SCALE, -point.Z() * SCALE );
+    
+                    point = trackPosition( t, 327.0 );
+                    if ( point.X() > -90 && point.Y() > -90 )
+                        vert( ofile, point.X() * SCALE, point.Y() * SCALE, -point.Z() * SCALE );
+    
+                    point = trackPosition( t, 349.5 );
+                    if ( point.X() > -90 && point.Y() > -90 )
+                        vert( ofile, point.X() * SCALE, point.Y() * SCALE, -point.Z() * SCALE );
+    
+                    point = trackPosition( t, 800.0 );
+                    if ( point.X() > -90 && point.Y() > -90 )
+                        vert( ofile, point.X() * SCALE, point.Y() * SCALE, -point.Z() * SCALE );
+    
```

Comment:
Looks like something for debugging. Can we remove it from the production code?

Reply:
removed